index.html

```diff
@@ -13,6 +13,27 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <!-- Qupte -->
+    <div class='quote-container' id='quote-container'>
+        <div class='quote-text'>
+            <i class='fas fa-quote-left'></i>
+            <span id='quote'>Don't judge each day by the harvest you reap but by the seeds that you plant.</span>
```

Comment:
aren't the quotes autogenerated?

Reply:
they are. in here i just wrote one to test the quote container and how it looks etc

Reply:
sounds good then, don't forget to remove the test later on